src/server/api/controllers/users.controller.js

```diff
@@ -18,6 +19,26 @@ const createUser = async (body) => {
   };
 };
 
+const getUsers = async () => knex('users');
+
+const getUserById = async (id) => {
+  if (!id) {
+    throw new HttpError('Bad request. Id should be a number', 400);
+  }
+
+  try {
+    const users = await knex('users')
+      .select('users.id as id', 'full_name', 'email', 'mobile', 'address')
+      .where({ id });
+    if (users.length === 0) {
+      throw new Error(`Specified ID does not existt`, 404);
+    }
+    return users;
+  } catch (error) {
+    return error.message;
```

Comment:
See my comment here https://github.com/HackYourFuture-CPH/fp-class19/pull/174/files#r811092971

Reply:
I changed it to what u wrote in the comments.